style/base.css

```diff
@@ -35,6 +35,10 @@ input::placeholder {
 .root {
   width: 100%;
   height: 100%;
```

Comment:
This messes up the toolbar location when there's a lot of video tiles. I think removing it and setting the height explicitly should keep the toolbar on the bottom.

Reply:
> setting the height explicitly

Do you mean setting the height of the root specifically, in pixels?

Reply:
So setting the `height` to `auto` on the root does not push the toolbar to the bottom, the height of the root becomes the height of its children